Formula/adios2.rb

```diff
@@ -0,0 +1,126 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "bzip2"
+  depends_on "c-blosc"
+  depends_on "catch2"
+  depends_on "hdf5"
+  depends_on "libfabric"
+  depends_on "libpng"
+  # mgard
+  depends_on "numpy"
+  # open-mpi
+  depends_on "python"
+  # sz
+  depends_on "zeromq"
+  # zfp
+
+  def install
+    args = std_cmake_args + %W[
+      -DADIOS2_USE_Blosc=ON
+      -DADIOS2_USE_BZip2=ON
+      -DADIOS2_USE_DataSpaces=OFF
+      -DADIOS2_USE_Fortran=ON
+      -DADIOS2_USE_HDF5=ON
+      -DADIOS2_USE_MGARD=OFF
+      -DADIOS2_USE_MPI=OFF
```

Comment:
@ax3l my preference would be to keep MPI ON

Reply:
yes works for me as well and I thought about this, too.

Reply:
I think we can add a `hdf5-mpi` and `adios2-mpi` formula later on as well.
@zbeekman do you know how we could add such additional formulas without much duplication and with proper conflicts? Can be done as a follow-up.